kategory/src/main/kotlin/kategory/collections/Maps.kt

```diff
@@ -0,0 +1,6 @@
+package kategory.collections
+
+import kategory.Tuple2
+
+fun <K, V> mapOf(vararg tuple: Tuple2<K, V>): Map<K, V> =
```

Comment:
The package should be just `package kategory`. I'm gonna approve it but please get it fixed before we merge it.